src/aics-image-viewer/components/TfEditor/index.tsx

```diff
@@ -13,56 +13,39 @@ import {
   ColorArray,
   colorArrayToObject,
   colorArrayToString,
-  ColorObject,
   colorObjectToArray,
 } from "../../shared/utils/colorRepresentations";
-import { Styles } from "../../shared/types";
+import { useRefWithSetter } from "../../shared/utils/hooks";
 
 export const TFEDITOR_DEFAULT_COLOR: ColorArray = [255, 255, 255];
-const COLOR_PICKER_MARGIN = 2;
-
-type Pair = [number, number];
+const TFEDITOR_COLOR_PICKER_MARGIN = 2;
+const TFEDITOR_GRADIENT_MAX_OPACITY = 0.9;
+const TFEDITOR_NUM_TICKS = 4;
+const TFEDITOR_MAX_BIN = 255;
```

Comment:
I'm actually not sure - it's possible that full dynamic range data could still be binned down into 256 bins. @toloudis may have a better sense though.

Reply:
Just replying to the discussion but not really adding any value to this PR:  Number of bins is not the same as max intensity.  This is one of the things we have to be careful about when moving to full dynamic range data.  So often you end up with having N bins, and if the bins are equally spaced then you have the first bin containing intensities in the set `[min, (min + (max-min)/N) )` and the second bin spans `[(min + (max-min)/N), (min + 2*(max-min)/N) )`,  and so on up to `[max- (max-min)/N), max)`. and because I wasn't precise enough, you have to know whether to use max or max+1, due to the open-ended upper bound on each bin. Then you have to decide how the lookup table maps to intensities based on the histogram binning bla bla bla